src/main/java/com/mparticle/kits/GoogleAnalyticsFirebaseKit.java

```diff
@@ -35,6 +35,13 @@
     final static String USER_ID_EMAIL_VALUE = "email";
     final static String USER_ID_MPID_VALUE = "mpid";
 
+    private static String[] forbiddenPrefixes = new String[]{"google_", "firebase_", "ga"};
```

Comment:
If you look into the Firebase docs its ga_ The TP issue just cut that off I believe.

Reply:
good catch, thought that one was a little out of place